File: fizzbuzz.lua
main = function ()
  local msg = ""
  for i = 1, 100 do
    if i % 5 == 0 and i % 3 == 0 then
      msg = "FizzBuzz"
    elseif i % 5 == 0 then
      msg = "Buzz"
    elseif i % 3 == 0 then
      msg = "Fizz"
    else
      msg = tostring(i)
    end
    print(i .. ": " .. msg)
  end
end

main()

File: fizzbuzz.js
function main() {
  let msg = "";
  for (let i = 1; i < 101; i++) {
    if (i % 5 == 0 && i % 3 == 0) {
      msg = "FizzBuzz";
    } else if (i % 5 == 0) {
      msg = "Buzz";
    } else if (i % 3 == 0) {
      msg = "Fizz";
    } else {
      msg = i.toString();
    }

    console.log(`${i}: ${msg}`);
  }
}
main();
